extensions/amp-story/1.0/amp-story.js

```diff
@@ -1524,6 +1614,13 @@ export class AmpStory extends AMP.BaseElement {
     return this.element;
   }
 
+  /**
+   * @return {string}
+   */
+  getId() {
+    return this.element.id || STANDALONE_STORY_ID;
```

Comment:
Just to clarify: `id` will be required at the validator level for any non standalone story?

Reply:
That's the idea; it's not currently implemented.

Reply:
Actually, I just removed the `getId` stuff entirely for now.  We can handle this if/when it's relevant.